Makefile

```diff
@@ -305,27 +305,26 @@ build-deps: manifests generate fmt
 .PHONY: build go-build-local
 build: build-deps go-build-local #HELP Build manager binary for current GOOS and GOARCH. Default target.
 go-build-local: BUILDBIN := bin
-go-build-local: $(BINARIES)
+go-build-local: $(BINARIES) go-build-catalogd-local
 
 .PHONY: build-linux go-build-linux
 build-linux: build-deps go-build-linux #EXHELP Build manager binary for GOOS=linux and local GOARCH.
 go-build-linux: BUILDBIN := bin/linux
 go-build-linux: export GOOS=linux
 go-build-linux: export GOARCH=amd64
-go-build-linux: $(BINARIES)
+go-build-linux: $(BINARIES) go-build-catalogd-linux
 
 .PHONY: run
-run: docker-build kind-cluster kind-load kind-deploy wait #HELP Build the operator-controller then deploy it into a new kind cluster.
+run: docker-build kind-cluster kind-load-catalogd kind-load kind-deploy wait #HELP Build the operator-controller then deploy it into a new kind cluster.
```

Comment:
we do not need new a new target
 kind-load will load both images

Reply:
This is combined.